ichnaea/data/tests/test_export.py

```diff
@@ -736,3 +766,15 @@ def test_datamap(self, celery, session):
         self._update_all(session, datamap_only=True)
         assert celery.data_queues["update_datamap_ne"].size() == 1
         assert celery.data_queues["update_datamap_sw"].size() == 1
+
+    def test_no_position(self, celery, session, metricsmock):
```

Comment:
This is the test for the new ``position: None`` functionality. Previously, this would raise an exception.